postinstall-win.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+
+if (process.platform === 'win') {
```

Comment:
This should be `win32`.
See https://nodejs.org/api/process.html#processplatform